content/docs/2.0/scalers/mysql.md

```diff
@@ -33,11 +33,11 @@ You can authenticate by using connection string or password authentication.
 
 **Connection String Authentication:**
 
-- `connection` - Connection string for MySQL database
+- `connectionFromEnv` - Connection string for MySQL database
```

Comment:
This feels a bit odd to me that we are renaming the auth parameters given FromEnv conflicts with pod identity, secrets,etc

Reply:
Sorry that I've missed this in the PR

Reply:
you are right, this shouldn't change.